client/src/auth/auth.js

```diff
@@ -1,9 +1,21 @@
+import axios from 'axios';
+
 const auth = {
   login(token, callback) {
-    localStorage.setItem('isAuthenticated', 'true'); // default true till server is setup.
-    console.log(token);
-    callback(); // This callback is only to spoof successful login till server is setup.
-    //             Will be moved to .then later
+    const data = {
```

Comment:
This is inconsistent with your server code. Which is extensible to handle all kinds of login. This one has hardcoded "google"